mantle/kola/tests/misc/multipath.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Red Hat, Inc.
```

Comment:
Completely unrelated to this but
https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated
(We could probably just drop the copyright statements since the year is a distraction)